pkg/urbit/daemon/main.c

```diff
@@ -362,6 +364,35 @@ _main_getopt(c3_i argc, c3_c** argv)
   return c3y;
 }
 
+/* u3_setup_cert_store: writes our embedded certificate database to a temp file
+ *
+ * Because OSX no longer ships with a real SSL certificate store, we have to
+ * embed our own trust store like every other system. Because we call a bunch
+ * of third party libraries, we write this to a temporary file and then set the
+ * environment variable about where the certificate store lives locally.
```

Comment:
I'm inclined to dispute this explanation. OS X ships with a real certificate store: the keychain. It's just only available through SecureTransport, but we're using OpenSSL. Mojava also appears to have shipped with LibreSSL (and a PEM-bundle), but I'm not sure if this is present by default or just included in the developer tools. Linux tends to have a bundle available, but the locations vary dramatically. So I'd say this is just about the inconsistent availability (and location) of PEM-encoded Root CA certificates on OS X and Linux.

Reply:
Deleted explanation.